app/models/generic_file.rb

```diff
@@ -10,6 +10,31 @@ def csv_headers
     ['id', 'mime_type', 'file_size', 'filename', 'checksum', 'depositor', 'title', 'resource_type', 'keyword', 'creator', 'rights', 'contributor', 'abstract', 'date_created', 'subject', 'language', 'identifier', 'location', 'related_url', 'provenance', 'in_collections']
   end
 
+  def admin_csv_headers
```

Comment:
Move to admin_csv_stuff Module?

Reply:
The csv headers for Collections and GenericFiles are different. So if we moved them out of the models we'd have to have two different methods, wouldn't we?
If it's in the model we can just pass the class into the `write_csv` method in the DownloadGenerator class and call:
`item_class.new.admin_csv_headers` and not have to check which type it is.

Reply:
Then keep the code as they are.